homeassistant/components/diagnostics/__init__.py

```diff
@@ -157,6 +161,23 @@ def handle_get(
     )
 
 
+@callback
+def async_format_manifest(manifest: Manifest) -> Manifest:
+    """Format manifest for diagnostics.
+
+    Remove the @ from codeowners so that
+    when users download the diagnostics and paste
+    the codeowners into the repository, it will
+    not notify the users in the codeowners file.
+    """
+    manifest_copy = manifest.copy()
+    if "codeowners" in manifest_copy:
+        manifest_copy["codeowners"] = [
+            codeowner.replace("@", "") for codeowner in manifest_copy["codeowners"]
```

Comment:
`lstrip` would be a bit more suitable
```suggestion
            codeowner.lstrip("@") for codeowner in manifest_copy["codeowners"]
```